src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -300,6 +310,36 @@ void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        void BeginMultipleConnections(Guid nodeId, int portIndex, PortType portType)
+        {
+            if (portType == PortType.Input) return; //only handle multiple connections when the port selected is an output port
+            activeStartPort = null;
+            var node = CurrentWorkspace.GetModelInternal(nodeId) as NodeModel;
+            if (node == null) return;
+
+            PortModel portModel = node.OutPorts[portIndex];
+
+            var models = new List<ModelBase>();
```

Comment:
Since models variable is to hold a list of connectors for deletion, you can rename it to connectorsForDeletion.